A/Arb/build_tarballs.jl

```diff
@@ -48,29 +45,28 @@ elif [[ ${target} == *mingw* ]]; then
    # MSYS_NT-6.3 is not detected as MINGW
    extraflags=--build=MINGW${nbits}
 fi
-
 ./configure --prefix=$prefix --disable-static --enable-shared --with-gmp=$prefix --with-mpfr=$prefix --with-flint=$prefix ${extraflags}
 make -j${nproc}
 make install LIBDIR=$(basename ${libdir})
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms(; experimental=build_for_julia16_or_newer)
+platforms = supported_platforms(; experimental=true)
 
 # The products that we will ensure are always built
 products = [
     LibraryProduct("libarb", :libarb)
 ]
 
 # Dependencies that must be installed before this package can be built
-v = build_for_julia16_or_newer ? v"200.800.101" : v"200.800.100"
+v = v"200.800.101"
 dependencies = [
     Dependency("FLINT_jll", v; compat = "~$v"),
```

Comment:
I think we can simplify this (and update to the latest version, just in case)
```suggestion
dependencies = [
    Dependency("FLINT_jll", compat = "~200.800.401"),
```